src/dds/rs-dds-device-proxy.cpp

```diff
@@ -490,4 +491,52 @@ void dds_device_proxy::hardware_reset()
 }
 
 
+std::vector< uint8_t > dds_device_proxy::send_receive_raw_data( const std::vector< uint8_t > & input )
+{
+    // debug_interface function
+    auto hexdata = rsutils::string::hexarray::to_string( input );
+    nlohmann::json control = nlohmann::json::object( { { "id", "hwm" }, { "data", hexdata } } );
+    nlohmann::json reply;
+    _dds_dev->send_control( control, &reply );
+    std::string default_status( "OK", 2 );
+    if( rsutils::json::get( reply, "status", default_status ) != default_status )
+        throw std::runtime_error( "Failed HWM: "
```

Comment:
Isn't there an `explanation`` field in case of failure?

Reply:
That's what it's supposed to be getting in the next line... I copy-pasted, maybe there's a bug somewhere else, too -- I'll check.